### ISSUE TITLE
Coding Style

### ISSUE THREAD

**Issue by dey4ss** [hyrise/sql-parser#192]
As we've seen [here](https://github.com/hyrise/sql-parser/pull/190#discussion_r710049905), the inconsistent use of tabs/spaces for indentation causes issues when reviewing PRs and is not nice in general.
Additionally, the tab/four space indentation is not in line with the `astyle.options` in the project root.
I suggest discussing the preferences in this issue. Afterward, I'm going to apply them in a new PR.

**Comment by Bensk1** [comment]
Good point. How about aligning it with the style used in hyrise/hyrise?
